otel-extension/src/main/java/io/scalac/mesmer/instrumentation/http/impl/SlashPathTemplateAdvice.java

```diff
@@ -0,0 +1,13 @@
+package io.scalac.mesmer.instrumentation.http.impl;
+
+import akka.http.scaladsl.server.PathMatcher;
+import io.opentelemetry.instrumentation.api.field.VirtualField;
+import net.bytebuddy.asm.Advice;
+
+public class SlashPathTemplateAdvice {
```

Comment:
(similar to above) looks like all (most of) template advices could have a generic advice that is parametrized